### PR TITLE
add router and price button

### PR DESCRIPTION
**Pull request by bhavesh-desai-scratch**

### CODE REVIEW

File: src/client/package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     ]
   },
   "dependencies": {
+    "@types/styled-system": "^2.3.2",
```
Comment (el-dav) [inline src/client/package.json]: should this be devDependancies?

File: src/client/src/rt-components/index.ts [hunk 2]
```diff
@@ -1,3 +1,4 @@
 export { EnvironmentProvider, withEnvironment, Environment } from './EnvironmentProvider'
 export { TearOff } from './tearOff'
 export { default as OpenFinChrome } from './OpenFinChrome'
+export { TradeButton, Flex, flexStyle } from './styleGuide'
```
Comment (el-dav) [inline src/client/src/rt-components/index.ts]: Were these files included in the commits? I can't see them

File: src/client/package.json [hunk 3]
```diff
@@ -89,6 +91,7 @@
     "reselect": "3.0.1",
     "rxjs": "6.1.0",
     "rxjs-spy": "^7.0.1",
+    "styled-system": "^3.0.2",
```
Comment (el-dav) [inline src/client/package.json]: I can't see styled-system used anywhere. Are you still using it?

File: src/client/package.json (hunk 1 above)
Comment (bhavesh-desai-scratch) [reply src/client/package.json]: Sorted. Deleted

File: src/client/src/rt-components/index.ts (hunk 2 above)
Comment (bhavesh-desai-scratch) [reply src/client/src/rt-components/index.ts]: Sorted. Deleted

**Pull request merged**